src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -1199,7 +1199,114 @@ export default class QuestionnaireForm extends Component {
       priorAuthBundle.signature = signature;
       priorAuthBundle.entry.unshift({ resource: priorAuthClaim });
 
+      const specialtyRxBundle = JSON.parse(JSON.stringify(priorAuthBundle));
```

Comment:
Is there a reason to keep the priorAuthBundle around?  Can we just remove it and use the specialtyRxBundle exclusively or does REMS have some use for the priorAuthBundle?

Reply:
I think its nice to show the synergy between the two but its ultimately not a part of this use case